### PR TITLE
Add HDFS (WebHDFS) Output Plugin article for v0.14

### PR DESCRIPTION
**Pull request by ryysud**
Referenced docs and source code:
- https://docs.fluentd.org/v0.14/articles/buffer-section
- https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin_helper/compat_parameters.rb
- https://github.com/fluent/fluent-plugin-webhdfs/blob/master/lib/fluent/plugin/out_webhdfs.rb

### CODE REVIEW

File: docs/v0.14/out_webhdfs.txt [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Path value can contain time placeholders. The following characters are replaced with actual values when the file is created:
+
+* %Y: year including the century (at least 4 digits)
+* %m: month of the year (01..12)
+* %d: Day of the month (01..31)
+* %H: Hour of the day, 24-hour clock (00..23)
+* %M: Minute of the hour (00..59)
+* %S: Second of the minute (00..60)
+
+## Time Sliced Output Parameters (and overwritten values by out_webhdfs)
```
Comment (repeatedly) [inline docs/v0.14/out_webhdfs.txt]: No `Time Sliced Output`

File: docs/v0.14/out_webhdfs.txt [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+Path value can contain time placeholders. The following characters are replaced with actual values when the file is created:
+
+* %Y: year including the century (at least 4 digits)
+* %m: month of the year (01..12)
+* %d: Day of the month (01..31)
+* %H: Hour of the day, 24-hour clock (00..23)
+* %M: Minute of the hour (00..59)
+* %S: Second of the minute (00..60)
+
```
Comment (repeatedly) [inline docs/v0.14/out_webhdfs.txt]: Add `<format>` is better.

**Comment by ryysud** [comment]
@repeatedly Thank you. I fix it.

**Comment by repeatedly** [comment]
Thanks!

**Pull request merged**